test/extensions/filters/network/tcp_proxy/tcp_proxy_test.cc

```diff
@@ -711,14 +711,14 @@ TEST_F(TcpProxyTest, UpstreamConnectTimeout) {
                     .value());
 
   filter_.reset();
-  EXPECT_EQ(access_log_data_, "UF");
+  EXPECT_EQ("UF", access_log_data_);
```

Comment:
I don't care about this one way or the other, but FYI: https://github.com/google/googletest/blob/master/googletest/docs/Primer.md says 

> Historical note: Before February 2016 *_EQ had a convention of calling it as ASSERT_EQ(expected, actual), so lots of existing code uses this order. Now *_EQ treats both parameters in the same way.

I think this is all over the place in the code base so not sure if it's worth swapping these and others in this PR?

Reply:
You are right; the language in EXPECT_EQ failure message used to assume the first arg was the expected value, and the second arg the actual.  Now the language is just kind of ambiguous, which IMO is a regression, but that does seem to be the standard.  Reverting these for this PR then.

I did also only overload the operator== in one direction to match what I thought was best practice, but I just added the other direction and it works fine.  Commit coming.